annotation-file-utilities/src/annotator/Main.java

```diff
@@ -866,6 +872,13 @@ public int compare(Pair<Integer, ASTPath> p1,
                   packageNames, toInsert);
               imports.addAll(packageNames);
             }
+            if (iToInsert instanceof AnnotationInsertion) {
+                AnnotationInsertion annoToInsert = (AnnotationInsertion) iToInsert;
```

Comment:
The rest of this file uses 2 spaces for indentation. Please adapt all your code to that style.

Reply:
Done.

Reply:
@CharlesZ-Chen Thanks for addressing the comments.
You don't have to write "Done" on every comment that you have addressed.  The reviewer will assume that you will respond to every comment (and when you do so, the original review comment will magically disappear).  You should respond only if you have a question or you disagree with the review.  This and some other advice about pull requests appears at:
https://homes.cs.washington.edu/~mernst/advice/github-pull-request.html#respond-to-reviews

Reply:
In general I agree with those PR response comments.
However, @CharlesZ-Chen is not in the typetools organization, so he cannot reassign the PR.
Also, sometimes a review comment needs changes in different locations and multiple files. Then "orphaned" comments hang around and coming back to a PR it is not clear that they have already been addressed. The "Done" message makes clear that the comment was addressed.

Github's UI could be better about this: when performing the review, I can comment as much as I want and then send one email with the review. The same should be possible for the response to a review. Instead of a "Done" email for each acknowledgement, I want one email when everything has been addressed.